docs/host/pypi.md

```diff
@@ -10,21 +10,90 @@ description: "The Open Web Calendar can be installed as a Python package from Py
 
 ![](https://img.shields.io/pypi/v/open-web-calendar.svg) ![](https://img.shields.io/pypi/pyversions/open-web-calendar.svg)
 
+## Installation
 The Open Web Calendar is available as a Python package on [PyPI]({{link.pypi}}).
 If you have [Python 3](https://www.python.org/) installed, run this to install the package:
 
 ```shell
 pip install open-web-calendar
 ```
+At this point you might get the error message
+```shell
+error: externally-managed-environment
+```
+
+On your productive system, it is strongly recommended to follow the instructions to create a virtual environment using 
+```shell
+python3 -m venv /home/username/venv
+```
+followed by 
+```shell
+/home/username/venv/bin/pip install open-web-calendar
+```
+This will install your open-web-calendar app in the project folder  
+/home/username/venv/lib/python3.12/site-packages/open_web_calendar
 
+## Execution
 After installation, run the Open Web Calendar using [Gunicorn](https://pypi.org/project/gunicorn/):
 
 ```shell
-gunicorn open_web_calendar:app
+/home/username/venv/bin/gunicorn open_web_calendar:app
 ```
 
 You should now see the server running at [http://127.0.0.1:8000](http://127.0.0.1:8000).
 
+However, in case you have installed the WebPage you are going to use for the integration of your individual calendar on the same local machine, it must be accessible to other machines as well.  
+In this case you might want to start the open_web_calendar with the dedicated IP-address of the hosting machine.
+
+```shell
+/home/username/venv/bin/gunicorn -b 192.168.178.7 open_web_calendar:app
+```
+You should now see the server running at [http://192.168.178.7:8000](http://192.168.178.7:8000) not only from your local machine.
+
+## Automatic Startup
+In order to start this service automatically at startup, you are required to create a systemd service script.
```

Comment:
yay!! cool

Reply:
I think, it would be nice to mention on which Operating System this one works and put it in the title and description.